Formula/rpm.rb

```diff
@@ -27,51 +13,40 @@ class Rpm < Formula
     sha256 "3a276b6a3f7273f8f88effbb057e8d65fc53fa9e812054951bfdd5ce618f54ac" => :yosemite
   end
 
-  depends_on "rpm2cpio" => :build
+  depends_on "beecrypt"
   depends_on "berkeley-db"
   depends_on "libmagic"
+  depends_on "libarchive"
```

Comment:
This dependency is usually redundant since macOS already provides libarchive.

Reply:
Unfortunately, macOS doesn't provide the headers, which means it fails to compile without it. :(

Reply:
ok